util/markup.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from dominate.tags import a
```

Comment:
Do you plan on making more use of this in the future, e.g. for manipulating the DOM? It seems overkill to add the library just to generate `<a href="...">...</a>`.